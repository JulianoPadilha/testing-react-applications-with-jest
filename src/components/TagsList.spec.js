import React from 'react';
import TagsList from './TagsList'; 
import renderer from 'react-test-renderer';

describe('The Tags List', () => {
  it('should render a snapshot', () => {
    const tree = renderer.create(<TagsList tags={['css', 'html', 'js']} />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});