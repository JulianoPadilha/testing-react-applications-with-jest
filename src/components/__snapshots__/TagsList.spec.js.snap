// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Tags List should render a snapshot 1`] = `
<div>
  <code>
    css
  </code>
  <code>
    html
  </code>
  <code>
    js
  </code>
</div>
`;
